libs/wingsdk/src/platform/platform-manager.ts

```diff
@@ -0,0 +1,85 @@
+import { basename, join } from "path";
+import { IPlatform } from "./platform";
+import { App, AppProps, SynthHooks } from "../core";
+
+interface PlatformManagerOptions {
+  readonly appProps: AppProps;
+  readonly platformPaths?: string[];
```

Comment:
We should note here that each value is either a path OR the name of a built-in platform